### PR TITLE
Story5

### PR DESCRIPTION
**Pull request by henrymeager**
Return book functionality has been added and styled to fit with mobile view

### CODE REVIEW

**Review by Andrei-Trifu — APPROVED** [review]
Great work! Looks good!

File: src/Components/BookDetail/ReturnBook/ReturnBook.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    const [name, setname] = useState('')
+
+    function emailChange(event) {
+        setEmail(event.target.value)
+    }
+
+    function submitBookReturn(event) {
+        event.preventDefault()
+    
+    fetch(`https://book-swap-api.dev.io-academy.uk/api/books/return/${bookID}`, {
```
Comment (ashleycoles) [inline src/Components/BookDetail/ReturnBook/ReturnBook.jsx]: Indention isn't quite right here

File: src/Components/BookDetail/ReturnBook/ReturnBook.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            headers: {
+                "Content-Type": "application/json",
+                "Accept": "application/json"
+            },
+            method: "PUT",
+            body: JSON.stringify({
+                'email': email
+            })
+        }).then((res) =>
+            {
```
Comment (ashleycoles) [inline src/Components/BookDetail/ReturnBook/ReturnBook.jsx]: This bracket should be on the same line as (res) =>

File: src/Components/BookDetail/ReturnBook/ReturnBook.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import { useState } from "react"
+import "./ReturnBook.css"
+
+function ReturnBook({ claimedByName, bookID, setClaimedByName}) {
+    
+    const [email, setEmail] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const [name, setname] = useState('')
```
Comment (ashleycoles) [inline src/Components/BookDetail/ReturnBook/ReturnBook.jsx]: Doesn't look like this name state is used, you should remove it if not

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Really nice work, very very happy with the way it works - just a few things to tidy up!

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**